src/storage-client/src/statistics.rs

```diff
@@ -489,6 +532,12 @@ impl SourceStatisticsUpdate {
         offset_known.incorporate(other.offset_known, "offset_known");
         offset_committed.incorporate(other.offset_committed, "offset_committed");
     }
+
+    /// Enrich statistics that use prometheus metrics.
+    pub fn with_metrics(mut self, metrics: &crate::metrics::StorageControllerMetrics) -> Self {
+        self.offset_known.0 .1 = Some(metrics.regressed_offset_known(self.id));
```

Comment:
`self.offset_known.0 .1 ` what's with the weird formatting?

Reply:
fixed with ur above suggestion!